core/app/models/spree/property.rb

```diff
@@ -48,7 +48,7 @@ def with_uniq_values_cache_key(product_properties_scope, &block)
     def ensure_product_properties_have_filter_params
       return unless filterable?
 
-      product_properties.where(filter_param: [nil, '']).where.not(value: [nil, '']).find_each(&:save)
```

Comment:
@aleksandarpetrushev please don't change this - this will break behaviour for existing installations with blank values